### PR TITLE
TASK: Remove deprecated methods

### PR DESCRIPTION
**Pull request by daniellienert**
Removes deprecated methods that were marked for remove
with Neos 4.0

Includes: 
Drop support for the .ts2 suffix especially the root.ts2 as fusion root file

### CODE REVIEW

**Comment by daniellienert** [comment]
@aertmann: Sorry Aske, while fixing the tests, a lot more deprecated TypoScript things surfaced. Would you please have another look?

**Comment by aertmann** [comment]
tests are failing due to the new UI package depending on the deprecated `HtmlAugmenter` instead of `FusionHtmlAugmenter` in https://github.com/neos/neos-ui/blob/850d21e89ddfad65cd358395f3299cf5e1489017/Classes/Neos/Neos/Ui/Aspects/NodeTypeSchemaCacheHeaderAspect.php#L19 and https://github.com/neos/neos-ui/blob/6cd4ec3949f96431fa96f4c5d026c6eee349f032/Classes/Neos/Neos/Ui/Aspects/AugmentationAspect.php#L22

**Comment by kitsunet** [comment]
https://github.com/neos/neos-ui/pull/1728 to fix the UI problem

**Comment by kitsunet** [comment]
Note, I am not sure if the change in the `Neos.Neos/Classes/Fusion/ContentElementWrappingImplementation.php` will not lead to problems with the old UI?

**Comment by aertmann** [comment]
Good catch @kitsunet! It definitely will. You mean the change in `ContentElementWrappingImplementation` right?

**Comment by kitsunet** [comment]
Yep, yep

File: Neos.Neos/Classes/Service/ContentElementWrappingService.php [hunk 1]
```diff
@@ -100,7 +99,6 @@ public function wrapCurrentDocumentMetadata(NodeInterface $node, $content, $fusi
         }
 
         $attributes = $additionalAttributes;
-        $attributes['data-node-__typoscript-path'] = $fusionPath; // @deprecated
```
Comment (aertmann) [inline Neos.Neos/Classes/Service/ContentElementWrappingService.php]: these two need to be part of #1937 instead

**Comment by kitsunet** [comment]
@aertmann I fixed it.

**Pull request merged**